cpp_common/package.xml

```diff
@@ -20,6 +20,9 @@
   <build_depend>boost</build_depend>
   <build_depend>libconsole-bridge-dev</build_depend>
 
-  <run_depend>boost</run_depend>
-  <run_depend>libconsole-bridge-dev</run_depend>
+  <exec_depend>boost</exec_depend>
+  <exec_depend>libconsole-bridge-dev</exec_depend>
```

Comment:
This should be removed, it should be either the system installed dependency (libconsole-bridge-dev) or the catkin installed one (console_bridge)

Reply:
Indeed. I will remove that.